pkg/debug/scripts/collect-info.sh

```diff
@@ -128,6 +133,21 @@ if nc -z -w 30 dl-cdn.alpinelinux.org 443 2>/dev/null; then
   apk add $PKG_DEPS >/dev/null 2>&1
 fi
 
+check_tar_flags() {
+  tar --version | grep -q "GNU tar"
+}
+
+collect_sysfs()
+{
+    echo "- sysfs"
+    local tarball_file="$DIR/sysfs.tar"
+    if check_tar_flags; then
+        tar --ignore-failed-read --warning=none -cf "$tarball_file" "/sys" > /dev/null 2>&1 || true
```

Comment:
Can we have issues with insufficient space in $DIR? If we do a czf here the file is 200x smaller.

Reply:
It increases the chance to run into insufficient space issue. I add the `z`.